lbuild/facade.py

```diff
@@ -265,49 +404,56 @@ class BuildLogFacade:
     def __init__(self, buildlog):
         self._buildlog = buildlog
 
-        self.__metadata = buildlog.metadata
-        self.__repo_metadata = buildlog.repo_metadata
-        self.__module_metadata = buildlog.module_metadata
-        self.__operation_metadata = buildlog.operation_metadata
-
     @property
     def outpath(self):
         return self._buildlog._outpath
 
     @property
+    @functools.lru_cache()
     def metadata(self):
-        return self.__metadata
+        deprecated("1.8.0", "BuildLog.metadata", EnvironmentBuildFacade(None).collect)
```

Comment:
Why do you add deprecation warnings and caching at the same time?

Reply:
Hm yes, the caching is overkill. Not sure what I was on there. 🤣

Reply:
Removed the caching for the better.